File: 1051-Height Checker-高度检查器/1051-Height Checker.java
class Solution {
    public int heightChecker(int[] heights) {
      int count = 0;
    	int newHeight[] = Arrays.copyOf(heights, heights.length);
      Arrays.sort(heights);
      for (int i = 0; i < newHeight.length; i++) {
        if(heights[i] != newHeight[i]) {
          count++;
        }
		  }
      return count;
    }
}

File: 1051-Height Checker-高度检查器/1051-Height Checker.js
/**
 * @param {number[]} heights
 * @return {number}
 */
var heightChecker = function(heights) {
    var oldHeight = heights.slice(0);
    heights.sort((a,b) => a-b)
    let  count = 0;
    for(let i = 0 ; i < heights.length ; i++){
      if(heights[i] != oldHeight[i]) 
      {
        count++;
      }
    }
    return count;
};
